src/components/datatable/DataTable.md

```diff
@@ -68,10 +111,12 @@ function Component() {
   };
 
   const generateRowId = (rowData) => {
+
     const [chapter] = rowData[2].split(delimiters.cell);
     const [verse] = rowData[3].split(delimiters.cell);
     const [uid] = rowData[4].split(delimiters.cell);
-    return `header-${chapter}-${verse}-${uid}`;
+    const rowId = `header-${chapter}-${verse}-${uid}`;
+    return rowId;
```

Comment:
Is there a reason for this change? Returning directly without declaring a variable is my preference but it really doesn't matter much.
```suggestion
    return `header-${chapter}-${verse}-${uid}`;
```